src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/ClinicalVariableCard.js

```diff
@@ -0,0 +1,533 @@
+                    : selectedSurvivalData}
+                  survivalPlotLoading={survivalPlotLoading}
+                  />
+              )}
+
+              {variable.active_chart === 'box' && (
+                <ClinicalBoxPlot
+                  boxPlotValues={boxPlotValues}
+                  cardFilters={cardFilters}
+                  dataBuckets={dataBuckets}
+                  fieldName={fieldName}
+                  qqData={qqData}
+                  setId={setId}
+                  setQQData={setQQData}
+                  setQQDataIsSet={setQQDataIsSet}
+                  theme={theme}
+                  totalDocs={totalDocs}
+                  type={variable.type}
+                  wrapperId={wrapperId}
+                  />
+              )}
+            </Column>
+
+            <Column>
+              <Row
+                style={{
+                  justifyContent: 'space-between',
+                  margin: '5px 0',
+                }}
+                >
+                <ActionsDropdown
+                  active_chart={variable.active_chart}
+                  cardFilters={cardFilters}
+                  currentAnalysis={currentAnalysis}
+                  dispatch={dispatch}
+                  selectedBins={selectedBins}
+                  styles={styles}
+                  theme={theme}
+                  totalDocs={totalDocs}
+                  tsvSubstring={tsvSubstring}
+                  />
+
+                {variable.active_chart === 'box' || (
+                  <Dropdown
+                    button={(
+                      <Button
+                        rightIcon={<DownCaretIcon />}
+                        style={{
+                          ...visualizingButton,
+                          padding: '0 12px',
+                        }}
+                        >
+                        Customize Bins
+                      </Button>
+                    )}
+                    dropdownStyle={{ right: 0 }}
+                    >
+                    <DropdownItem
+                      onClick={openCustomBinModal}
+                      style={styles.actionMenuItem}
+                      >
+                      Edit Bins
+                    </DropdownItem>
+                    <DropdownItem
+                      onClick={resetBins}
+                      style={{
+                        ...styles.actionMenuItem,
+                        ...binsAreCustom
+                          ? {}
+                          : styles.actionMenuItemDisabled(theme),
+                      }}
+                      >
+                      Reset to Default
+                    </DropdownItem>
+                  </Dropdown>
+                )}
+              </Row>
+
+              <EntityPageHorizontalTable
+                data={tableData.map(tableRow => Object.assign(
+                  {},
+                  tableRow,
+                  {
+                    key: variable.plotTypes === 'categorical'
```

Comment:
Pass a prop (for example `tableRowName`) and assign it to this key instead of checking plotTypes.

Reply:
added another property to `displayData` so I don't have to check for `plotTypes` or `groupName` all the time :+1: